pep-0630.rst

```diff
@@ -322,23 +339,27 @@ the Python level: for example, you can't set ``str.myattribute = 123``.
 
 .. note::
    Sharing truly immutable objects between interpreters is fine,
-   as long as they don't provide access to mutable objects. But, every
-   Python object has a mutable implementation detail: the reference
-   count. Changes to the refcount are guarded by the GIL. Thus, code
-   that shares any Python objects across interpreters implicitly depends
-   on CPython's current, process-wide GIL.
+   as long as they don't provide access to mutable objects.
+   However, in CPython, every Python object has a mutable implementation
+   detail: the reference count. Changes to the refcount are guarded by the GIL.
+   Thus, code that shares any Python objects across interpreters implicitly
+   depends on CPython's current, process-wide GIL.
+
+   :pep:`683` proposes a mechanism to mark certain objects as
+   immortal, such that they will be truly immutable, though is is likewise
+   considered an internal implementation detail.
```

Comment:
I don't want to put this in the HOWTO. There's a lot of details this note is omitting, and a proposal to change some of them isn't particularly relevant.
```suggestion
```
(PEP 684 (A Per-Interpreter GIL) will affect extension authors, but that documentation should be added when the PEP is implemented.)

Reply:
Okay; I added this since you refer to a number related draft and approved PEPs elsewhere in the document, and in the context that the target of this PR is still a PEP (and presumably, will remain on this repo in this state indefinitely, presumably marked "Final" and with a prominent admonition redirecting users to the current canonical HOWTO).

Reply:
```suggestion
```